Running the fuzzers in CI error out

See e.g. [this recent run](https://github.com/uutils/diffutils/actions/runs/8773238273/job/24073220530):

```
     Running `fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal -artifact_prefix=/home/runner/work/diffutils/diffutils/fuzz/artifacts/fuzz_normal/ -max_total_time=60 -detect_leaks=0 /home/runner/work/diffutils/diffutils/fuzz/corpus/fuzz_normal`
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 3739805489
INFO: Loaded 1 modules   (44194 inline 8-bit counters): 44194 [0x564b92fe4f70, 0x564b92fefc12), 
INFO: Loaded 1 PC tables (44194 PCs): 44194 [0x564b92fefc18,0x564b9309c638), 
INFO:        0 files found in /home/runner/work/diffutils/diffutils/fuzz/corpus/fuzz_normal
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
thread '<unnamed>' panicked at fuzz_targets/fuzz_normal.rs:27:10:
called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
==2697== ERROR: libFuzzer: deadly signal
    #0 0x564b92bd87c1  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x1ef7c1) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #1 0x564b92ddf52e  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3f652e) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #2 0x564b92dd5939  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3ec939) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #3 0x7fb2cac4251f  (/lib/x86_64-linux-gnu/libc.so.6+0x4251f) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #4 0x7fb2cac969fb  (/lib/x86_64-linux-gnu/libc.so.6+0x969fb) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #5 0x7fb2cac42475  (/lib/x86_64-linux-gnu/libc.so.6+0x42475) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #6 0x7fb2cac287f2  (/lib/x86_64-linux-gnu/libc.so.6+0x287f2) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #7 0x564b92e8a449  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x4a1449) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #8 0x564b92b332e9  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x14a2e9) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #9 0x564b92dcf5e4  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3e65e4) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #10 0x564b92e7f2b5  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x4962b5) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #11 0x564b92e7f063  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x496063) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #12 0x564b92e7c9a8  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x4939a8) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #13 0x564b92e7ed96  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x495d96) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #14 0x564b92b35832  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x14c832) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #15 0x564b92b35dc5  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x14cdc5) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #16 0x564b92c227c5  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x2397c5) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #17 0x564b92c2133a  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x23833a) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #18 0x564b92dca50f  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3e150f) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #19 0x564b92dcf7f7  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3e67f7) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #20 0x564b92dced1b  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3e5d1b) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #21 0x564b92dd5e85  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3ece85) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #22 0x564b92dde052  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3f5052) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #23 0x564b92dde6b7  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x3f56b7) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #24 0x564b92dfa55f  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x41155f) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #25 0x564b92b364b6  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x14d4b6) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)
    #26 0x7fb2cac29d8f  (/lib/x86_64-linux-gnu/libc.so.6+0x29d8f) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #27 0x7fb2cac29e3f  (/lib/x86_64-linux-gnu/libc.so.6+0x29e3f) (BuildId: c289da5071a3399de893d2af81d6a30c62646e1e)
    #28 0x564b92b36614  (/home/runner/work/diffutils/diffutils/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_normal+0x14d614) (BuildId: 40fbff1e52749f340ec245969cf12ef48ea09d79)

NOTE: libFuzzer has rudimentary signal handlers.
      Combine libFuzzer with AddressSanitizer or similar for better crash reports.
SUMMARY: libFuzzer: deadly signal
MS: 0 ; base unit: 0000000000000000000000000000000000000000


artifact_prefix='/home/runner/work/diffutils/diffutils/fuzz/artifacts/fuzz_normal/'; Test unit written to /home/runner/work/diffutils/diffutils/fuzz/artifacts/fuzz_normal/crash-da39a3ee5e6b4b0d3255bfef95601890afd80709
Base64: 

────────────────────────────────────────────────────────────────────────────────

Failing input:

	fuzz/artifacts/fuzz_normal/crash-da39a3ee5e6b4b0d3255bfef95601890afd80709

Output of `std::fmt::Debug`:

	(
	    [],
	    [],
	)

Reproduce with:

	cargo fuzz run fuzz_normal fuzz/artifacts/fuzz_normal/crash-da39a3ee5e6b4b0d3255bfef95601890afd80709

Minimize test case with:

	cargo fuzz tmin fuzz_normal fuzz/artifacts/fuzz_normal/crash-da39a3ee5e6b4b0d3255bfef95601890afd80709

────────────────────────────────────────────────────────────────────────────────

Error: Fuzz target exited with exit status: 77
Error: Process completed with exit code 1.
```

--- a/.github/workflows/fuzzing.yml
+++ b/.github/workflows/fuzzing.yml
@@ -68,7 +68,7 @@ jobs:
             fuzz/corpus/${{ matrix.test-target.name }}
       - name: Run ${{ matrix.test-target.name }} for XX seconds
         shell: bash
-        continue-on-error: ${{ !matrix.test-target.name.should_pass }}
+        continue-on-error: ${{ !matrix.test-target.should_pass }}
         run: |
           cargo fuzz run ${{ matrix.test-target.name }} -- -max_total_time=${{ env.RUN_FOR }} -detect_leaks=0
       - name: Save Corpus Cache

--- a/fuzz/fuzz_targets/fuzz_cmp.rs
+++ b/fuzz/fuzz_targets/fuzz_cmp.rs
@@ -4,7 +4,7 @@ extern crate libfuzzer_sys;
 use diffutilslib::cmp::{self, Cmp};
 
 use std::ffi::OsString;
-use std::fs::File;
+use std::fs::{self, File};
 use std::io::Write;
 
 fn os(s: &str) -> OsString {
@@ -18,7 +18,7 @@ fuzz_target!(|x: (Vec<u8>, Vec<u8>)| {
         .peekable();
 
     let (from, to) = x;
-
+    fs::create_dir_all("target").unwrap();
     File::create("target/fuzz.cmp.a")
         .unwrap()
         .write_all(&from)

--- a/fuzz/fuzz_targets/fuzz_cmp_args.rs
+++ b/fuzz/fuzz_targets/fuzz_cmp_args.rs
@@ -11,6 +11,9 @@ fn os(s: &str) -> OsString {
 }
 
 fuzz_target!(|x: Vec<OsString>| -> Corpus {
+    if x.iter().any(|a| a == "--help") {
+        return Corpus::Reject;
+    }
     if x.len() > 6 {
         // Make sure we try to parse an option when we get longer args. x[0] will be
         // the executable name.

--- a/fuzz/fuzz_targets/fuzz_ed.rs
+++ b/fuzz/fuzz_targets/fuzz_ed.rs
@@ -38,6 +38,7 @@ fuzz_target!(|x: (Vec<u8>, Vec<u8>)| {
     } else {
         return;
     }
+    fs::create_dir_all("target").unwrap();
     let diff = diff_w(&from, &to, "target/fuzz.file").unwrap();
     File::create("target/fuzz.file.original")
         .unwrap()

--- a/fuzz/fuzz_targets/fuzz_normal.rs
+++ b/fuzz/fuzz_targets/fuzz_normal.rs
@@ -23,6 +23,7 @@ fuzz_target!(|x: (Vec<u8>, Vec<u8>)| {
         return
     }*/
     let diff = normal_diff::diff(&from, &to, &Params::default());
+    fs::create_dir_all("target").unwrap();
     File::create("target/fuzz.file.original")
         .unwrap()
         .write_all(&from)

--- a/fuzz/fuzz_targets/fuzz_patch.rs
+++ b/fuzz/fuzz_targets/fuzz_patch.rs
@@ -21,6 +21,7 @@ fuzz_target!(|x: (Vec<u8>, Vec<u8>, u8)| {
     } else {
         return
     }*/
+    fs::create_dir_all("target").unwrap();
     let diff = unified_diff::diff(
         &from,
         &to,

--- a/fuzz/fuzz_targets/fuzz_side.rs
+++ b/fuzz/fuzz_targets/fuzz_side.rs
@@ -4,7 +4,7 @@ extern crate libfuzzer_sys;
 
 use diffutilslib::side_diff;
 
-use std::fs::File;
+use std::fs::{self, File};
 use std::io::Write;
 use diffutilslib::params::Params;
 
@@ -21,6 +21,7 @@ fuzz_target!(|x: (Vec<u8>, Vec<u8>, /* usize, usize */ bool)| {
         expand_tabs: expand,
         ..Default::default()
     };
+    fs::create_dir_all("target").unwrap();
     let mut output_buf = vec![];
     side_diff::diff(&original, &new, &mut output_buf, &params);
     File::create("target/fuzz.file.original")

